modules/programs/git.nix

```diff
@@ -76,6 +79,15 @@ in
         description = "Default user email to use.";
       };
 
+      sendemail.identity = mkOption {
+        type = types.nullOr types.str;
+        default = let
```

Comment:
Wouldn't this cause a documentation rebuild if the user changes which email account is primary?

I think its best to just remove this option entirely and instead always use the primary account. If a user wants to override it then they'll be able to set it directly through `extraConfig` (once I fix the type after merging this PR).

Reply:
you are right let's keep it simple, extraConfig should be fine. I pushed an update that removes this option. Feel free to cherry-pick/squash/complete it.